<launch>
  <node output="screen" pkg="pozyx_ros_driver" name="pozyx" type="pozyx.py" clear_params="true">
    <rosparam command="load" file="$(find pozyx_ros_driver)/launch/main.yaml"/>
    <param name="rate" value='10.0'/>
    <param name="enable_position" value="true"/>
    <param name="enable_orientation" value="false"/>
    <param name="enable_sensors" value="false"/>
    <param name="position_stddev" value="0.25"/>
    <param name="linear_acceleration_stddev" value="0.1"/>
    <param name="angular_velocity_stddev" value="0.005"/>
    <param name="magnetic_field_stddev" value="0.000002"/>
    <param name="orientation_stddev" value="0.001"/>
    <param name="debug" value="false"/>
    <param name="continuous" value="false"/>
  </node>
</launch>
